vendor/forms.py

```diff
@@ -10,19 +10,30 @@
 
 from .config import VENDOR_PRODUCT_MODEL
 from .models import Address, Offer, Price, offer_term_details_default
-from .models.choice import PaymentTypes, TermType
+from .models.choice import PaymentTypes, TermType, Country
 from .utils import get_site_from_request
 
 Product = apps.get_model(VENDOR_PRODUCT_MODEL)
 
-        
+COUNTRY_CHOICE = getattr(settings, 'VENDOR_COUNTRY_CHOICE', Country)
+COUNTRY_DEFAULT = getattr(settings, 'VENDOR_COUNTRY_DEFAULT', Country.US)
+
```

Comment:
Can you just do settings.VENDOR_COUNTRY_CHOICE?

Reply:
I guess this is just to test, but this seems like a per-site setting, not a global env var setting.

Reply:
Don't understand the question jejeje?

The idea is that if there is no `VENDOR_COUNTRY_CHOICE` the `COUNTRY_CHOICE` uses the complete list. Otherwise use the defined list